docker-scripts/klaw-docker.sh

```diff
@@ -21,6 +21,16 @@ build () {
 
 }
 
+buildForUITests () {
+	echo "Build Klaw project binaries"
+	mvn mvn clean install -Dmaven.test.skip=true
```

Comment:
The double mvn here is causing the build to fail it should be 
mvn clean install -Dmaven.test.skip=true

Reply:
OH GOSH 😂  I didn't notice!